<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	font: 81 bold 20pt &quot;Fira Sans Compressed&quot;;
	border: 1px;
}

QPushButton {
	background-color: 'yellow';
	font: 81 bold 20pt &quot;Fira Sans Compressed&quot;;
	border-radius: 6px;
}

#playerName, #casinoName, #cardsLabel, #valuesLabel, #pointsLabel {
	background-color: red;
	border: 1px solid 'black';
}

#Window {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 74, 0, 255), stop:0.739796 rgba(0, 192, 0, 255));
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="1" column="1">
      <widget class="QWidget" name="playerCardsHandler" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>10</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QHBoxLayout" name="playerCardsLayout"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="buttonsLayout">
       <item>
        <widget class="QPushButton" name="hitButton">
         <property name="text">
          <string>Hit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="standButton">
         <property name="text">
          <string>Stand</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="playerValues">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="playerPoints">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="casinoName">
       <property name="text">
        <string>Casino</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QWidget" name="casinoCardsHandler" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>3</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="casinoCardsLayout"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="cardsLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Cards</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="valuesLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Values</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="casinoValues">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="casinoPoints">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="pointsLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Points</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="playerName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>You</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
